lib/archethic/bootstrap.ex

```diff
@@ -207,13 +209,83 @@ defmodule Archethic.Bootstrap do
       Logger.info("Synchronization finished")
     end
 
+    resync_network_chain([:oracle, :node_shared_secrets], P2P.authorized_and_available_nodes())
+
     Sync.publish_end_of_sync()
     SelfRepair.start_scheduler()
 
     :persistent_term.put(:archethic_up, :up)
     Listener.listen()
   end
 
+  # when no node loaded in memtable
+  @spec resync_network_chain(any, maybe_improper_list) :: :ok
```

Comment:
Why did you mention it as improper list ?
Also the list of types would every time be the same so the first parameter is not required I think